dell/src/test/java/org/apache/iceberg/dell/mock/ecs/TestExceptionCode.java

```diff
@@ -66,12 +67,17 @@ public void testExceptionCode() {
   }
 
   public void assertS3Exception(String message, int httpCode, String errorCode, Runnable task) {
-    try {
-      task.run();
-      Assert.fail("Expect s3 exception for " + message);
-    } catch (S3Exception e) {
-      Assert.assertEquals(message + ", http code", httpCode, e.getHttpCode());
-      Assert.assertEquals(message + ", error code", errorCode, e.getErrorCode());
-    }
+    Assertions.assertThatThrownBy(task::run)
```

Comment:
This seems more complicated than the original try/catch. Not sure it's worth it.